### PR TITLE
Removing old analysis functionality

### PR DESCRIPTION
**Pull request by ManInFez**
Removing functionality in analysis step only used by removed update implementations

**Issue**
Resolves #my_issue

**Approach**
_Short description of the approach_

### CODE REVIEW

**Review by oyvindeide — COMMENTED** [review]
Looks good, some minor comments, also could you add some of the context you have about for example `module_data_block_vector` in the commit text?

File: libres/lib/enkf/tests/gen_kw_test.cpp [hunk 1]
```diff
@@ -21,6 +21,7 @@
 #include <stdlib.h>
 #include <stdio.h>
 
+#include <ert/util/vector.hpp>
```
[the remark this answers is not in the document]
Comment (ManInFez) [reply libres/lib/enkf/tests/gen_kw_test.cpp:24]: Compiler complained if I didnt include this

File: libres/lib/enkf/enkf_main.cpp [hunk 2]
```diff
@@ -24,6 +24,7 @@
 
 #define HAVE_THREAD_POOL 1
 #include <ert/util/rng.h>
+#include <ert/util/vector.hpp>
```
[the remark this answers is not in the document]
Comment (ManInFez) [reply libres/lib/enkf/enkf_main.cpp:27]: Compiler complained if I didnt include this

**Comment by ManInFez** [comment]
Test ert please!

**Comment by ManInFez** [comment]
test ert please!

**Review by oyvindeide — APPROVED** [review]
LGTM! Thanks for cleaning 😄

**Pull request merged**